CrustTests/CRMapperTests.swift

```diff
@@ -21,7 +21,8 @@ class MockMap: Mapping, Adapter {
         catchMapping!(toMap, context)
     }
     
-    func mappingBegins() throws { }
+    var mappingDidBegin: Bool = false
+    func mappingBegins() throws { self.mappingDidBegin = true }
```

Comment:
Want to fix this naming convention while you're at it?

It was mappingBegan/mappingEnded, from what we talked about, no?